examples/CMakeLists.txt

```diff
@@ -22,13 +22,18 @@ endif()
 
 set(TARGET common)
 
+if (WHISPER_FFMPEG)
+    set(FFMPEG_SRC ffmpeg_transcode.cpp)
+endif()
+
 add_library(${TARGET} STATIC
     common.h
     common.cpp
     common-ggml.h
     common-ggml.cpp
     grammar-parser.h
     grammar-parser.cpp
+    ${FFMPEG_SRC}
```

Comment:
For consistency, rename this variable to `COMMON_SOURCES_FFMPEG`

Reply:
done